### PR TITLE
Add vagrant plugin install helper notes

### LINKED ISSUE #23: Vagrant plugin install issues on Fedora 33
Part of the host provision procedure is installation of required vagrant plugins.
On fresh Fedora 33 plugin install ends up with the error:
```
conflicting dependencies racc (= 1.4.16) and racc (= 1.5.0)
  Activated racc-1.5.0
  which does not match conflicting dependency (= 1.4.16)

  Conflicting dependency chains:
    racc (= 1.5.0), 1.5.0 activated

  versus:
    racc (= 1.4.16)

  Gems matching racc (= 1.4.16):
    racc-1.4.16
```

This has been already addressed in BZ [1]. For me workaround was to explicit disable dependency version checking in vagrant:
```
$ VAGRANT_DISABLE_STRICT_DEPENDENCY_ENFORCEMENT=1 ./sssd-test-suite provision host --pool /vm-pool/
```

I am wondering if we can address similar issues for the future enabling some more liberal plugins / dependencies checking by default in `sssd-test-suite`.

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1835308

### PR DESCRIPTION
**Pull request by elkoniu**
In case of vagrant plugins installation failure small
howto will be displayed. It describes some common
issues happening during vagrant plugin install.

Closes https://github.com/SSSD/sssd-test-suite/issues/23

### CODE REVIEW

**Comment by pbrezina** [comment]
I will probably create a copr repository with winrm and winrm-elevated packages. This should fix the issue forever.

**Comment by elkoniu** [comment]
> I will probably create a copr repository with winrm and winrm-elevated packages. This should fix the issue forever.

This way we will be always using fixed and specific version of plugins? So this will be sort-of plugins proxy?

**Comment by pbrezina** [comment]
The problem is that we mix distribution rpms with native gems so we end up with dependency issues every now and then. If we built it in copr, it will use and require only packaged stuff and it should work all the time without workarounds.

**Comment by elkoniu** [comment]
@pbrezina  Please advice if we should have this helper message meanwhile (before copr build) or we should drop this PR? For me personally setting sssd-test-suite on new fedora is frequently googling around those broken vagrant dependencies and I need to remind myself that they can be skipped by force - which usually works fine for me.

File: provision/roles/host/tasks/Fedora.yml [hunk 1]
```diff
@@ -61,3 +62,25 @@
...
+      "   $ VAGRANT_DISABLE_STRICT_DEPENDENCY_ENFORCEMENT=1 ./sssd-test-suite provision host --pool $VM_POOL"
+      ""
+      "2) In case vagrant has been updated recently please follow \"vagrant plugin install\" stdout suggestions:"
+      "   $ vagrant plugin repair"
+      "   $ vagrant plugin expunge --reinstall"
+      "   This can be combined with (1)"
+      ""
+      "3) Worst case scenario may reguire removal of \".vagrant\" and \".vagrant.d\" directories from project"
+      "   directory and user $HOME".
+      "------------------------------------------------------------------------------------------------------"
```
Comment (pbrezina) [inline provision/roles/host/tasks/Fedora.yml:84]: Are these quotes really required? Using | already denotes a multiline string.

Comment (elkoniu) [reply provision/roles/host/tasks/Fedora.yml:84]: Fixed

**Comment by elkoniu** [comment]
@pbrezina thank you for the review, I have updated the code to remove additional quotes. Please take a look now.

**Comment by elkoniu** [comment]
@pbrezina thank you for the review, I have updated the code to remove additional quotes. Please take a look now.

**Comment by elkoniu** [comment]
I guess this one is deprecated already by now - closing

**Pull request closed without merging**